proposals/0253-callable.md

```diff
@@ -3,8 +3,9 @@
 * Proposal: [SE-0253](https://github.com/apple/swift-evolution/blob/master/proposals/0253-callable.md)

```

Comment:
The proposal name is very generic, and Swift already has callable things.  How about "User Defined Callable Types"?

Reply:
I'm intentionally avoiding "callable types" because 1. `func call` is not restricted to primary type declarations and there's no type declaration attributes, and 2. _types_ being callable sound like initializers.

Reply:
I changed the title to "User-defined callable values". What do you think?

Reply:
Isn't fp a user defined callable value?
```
var fp : () -> () = {}
```

Reply:
That's a good point. How about "non-function callable values"?

Reply:
Metatypes.

Why not call this "Allowing user defined types to be callable"?

Reply:
Metatypes are not callable today actually, though I understand in the implementation they are.

```swift
let x = Float.self
x(2)
```
```console
error: repl.swift:2:2: error: initializing from a metatype value must reference 'init' explicitly
x(2)
 ^
 .init
```

Reply:
I changed it to "Callable values of user-defined nominal types" to be a bit more accurate. Hope that sounds okay to you!